jenkins/job-configs/jenkins-daily-maintenance.yaml

```diff
@@ -63,3 +63,29 @@
                 </hudson.plugins.parameterizedtrigger.TriggerBuilder>
     triggers:
         - timed: '@daily'
+
+- job-template:
+    name: 'google-cloud-resource-cleanup'
+    description: 'Clean up leaked cloud resources. Test owner: test-infra maintainers.'
+    properties:
+        - build-discarder:
+            days-to-keep: 30
+    scm:
+    - git:
+        url: https://github.com/kubernetes/test-infra
+        branches:
+        - master
+        browser: githubweb
+        browser-url: https://github.com/kubernetes/test-infra
+        skip-tag: true
+    triggers:
+        - timed: '@daily'
```

Comment:
We want to block the SQ when things are broken, and if they're leaking so fast that they run out of quota in a day then they're broken.